dom0-updates/qubes-dom0-update

```diff
@@ -234,6 +234,43 @@ if [ -n "$CLEAN" ]; then
 fi
 rm -f /var/lib/qubes/updates/errors
 
+# Synopsis: check_mounted MOUNTPOINT
+check_mounted() {
+    local CHOICE
+    # No reason to check further if mount point is already mounted
+    awk -v PART="${1}" '{if ($2 == PART ) { exit 0 }} ENDFILE{exit -1}' < /proc/mounts
+    [[ ${?} -ne 0 ]] || return
+    # No reason to check further if mount point is not in fstab
+    awk -v PART="${1}" '!/^[ \t]*#/{ if ( $2 == PART ) { exit 0}} ENDFILE {exit -1}' < /etc/fstab
+    [[ ${?} -ne 0 ]] && return
+    # Ask user to manually mount partition if user is using GUI Updater
+    if [ ! -t 1 ]; then
+        echo "Could not decide about unmounted ${1} partition in non-interactive/GUI mode!"
```

Comment:
This doesn't allow overriding the check in non-interactive case. Maybe add some option for that?

Reply:
> This doesn't allow overriding the check in non-interactive case. Maybe add some option for that?

Is `--skip-boot` or `--skip-boot-check` good? And the help text:

```
--skip-boot       do no check if /boot & /boot/efi are mounted.
```